### DISCUSSION TITLE
Where is x-instagram-ajax value?

### DISCUSSION THREAD

**Discussion by Argusf28** [chriskyfung/AutoFetcher-IG-Stories-to-GDrive#70] [Q&A] [answered]
The fetcher was working just fine until December 22nd.
I replaced some apparently old values, but I cant find "x-instagram-ajax" nor do I know if that is the problem.

**Comment by chriskyfung** [comment, marked as answer]
🧹 Deprecated `x-instagram-ajax` from Build2023.02.14 ( See Merge #73 )

**Reply by chriskyfung** [reply]
Same as Issue #68
